src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -256,7 +256,7 @@ export const EmailIdentityAuthEntry = React.createClass({
         } else {
             return (
                 <div>
-                    <p>{ _t("An email has been sent to") } <i>{ this.props.inputs.emailAddress }</i></p>
+                    <p>{ _t("An email has been sent to %(emailAddress)s", { emailAddress: '<i>' + this.props.inputs.emailAddress + '</i>' }) }</p>
```

Comment:
Won't this write the string `<i>`?

Reply:
I think too there needs to be _tJsx used